js/classes.js

```diff
@@ -253,21 +271,43 @@ class UserShip extends Ship {
 
 class EnemyShip extends Ship {
   constructor(shape, name, health) {
-    super(shape, name, health)
+    super(shape, name, null, health)
```

Comment:
Why the addition of `null` here?

Reply:
Changing to accept color. It's cuz `EnemyShip` doesn't aceppt a color but Ship does  (https://github.com/michaelwschultz/space-shooter-led/pull/12/files#diff-c98a8a6f8fc25f2f4c08ff43eb922499R107)